SiteLogToGeodesyML/sitelogtogeodesyml.py

```diff
@@ -1429,9 +1433,10 @@ class SiteIdentification(object):
     @classmethod
     def Detach(cls):
         if cls.Current:
-            identification = cls.Current.complete()
+            wrapper = geo.SiteIdentificationPropertyType()
+            wrapper.append(cls.Current.complete())
             cls.Current = None
-            return identification
+            return wrapper
```

Comment:
Would it be simpler to leave this `Detach` function unmodified, but to instead make an equivalent change at the invocation site,

from:

`self.siteLog.siteLocation = SiteLocation.Detach()`

to

`self.siteLog.siteLocation = geo.SiteLocationPropertyType(SiteLocation.Detach())`

Would that change be equivalent? It's less work and it doesn't make `SiteLocation` class depend on `SiteLocationPropertyType`.

Reply:
Changes have been done as suggested. It works fine and looks better.